src/main/resources/static/_shared.yml

```diff
@@ -2469,9 +2469,6 @@ components:
           type: string
         assessorName:
           type: string
-      required:
```

Comment:
I'll probably squash this into the original commit where it was introduced, but @Harriethw is there a way we tend to test the non-requiredness of these fields or do you think this behaviour is covered by the fact we're using OpenAPI-generated code and the existing integration/unit tests?

Reply:
hmm actually thinking about it today, I think we need to go a different approach and make the fields required, but set `nullable: true`.
When this model is used in the Service it treats it as a replacement for all the existing data and overwrites both fields, and if we make the fields not required I'm concerned that implies to the frontend if it doesn't supply one of the fields then it won't be updated?

Reply:
We've discovered this doesn't work as expected, so we've removed `nullable: true` but we're allowing submission of the assessment keys with an empty string value.